src/vs/editor/contrib/hover/browser/modesContentHover.ts

```diff
@@ -245,24 +242,7 @@ export class ModesContentHoverWidget extends ContentHoverWidget {
 			msg.contents
 				.filter(contents => !!contents)
 				.forEach(contents => {
-					const renderedContents = renderMarkedString(contents, {
-						actionCallback: (content) => {
-							this._openerService.open(URI.parse(content)).then(void 0, onUnexpectedError);
-						},
-						codeBlockRenderer: (languageAlias, value): string | TPromise<string> => {
-							// In markdown,
-							// it is possible that we stumble upon language aliases (e.g.js instead of javascript)
-							// it is possible no alias is given in which case we fall back to the current editor lang
-							const modeId = languageAlias
-								? this._modeService.getModeIdForLanguageName(languageAlias)
-								: this._editor.getModel().getModeId();
-
-							return this._modeService.getOrCreateMode(modeId).then(_ => {
-								return `<div class="code">${tokenizeToString(value, modeId)}</div>`;
-							});
-						}
-					});
-
+					const renderedContents = renderMarkedString(this._modeService, this._openerService, contents);
```

Comment:
@jrieken I tried to enumerate the different input/output cases in my [comment here](https://github.com/Microsoft/vscode/pull/16503#issuecomment-265023706), but it sounds like you have another case in mind other than the ones enumerated there? If so, would you mind clarifying what the expected input and output is? I can't seem to construct a case where `content.code`gets set, but the `languageAlias` is not set, at least not as the `MarkedString` type is defined. 
```ts
const htmlContentElement = typeof markedString === 'string' ? { markdown: markedString } : { code: markedString };
```